ibis-server/tools/query_local_run.py

```
@@ -74,5 +75,12 @@
     print("### Result ###")
     print("")
     print(df)
+elif data_source == "mysql":
+    connection_info = MySqlConnectionInfo.model_validate_json(json.dumps(connection_info))
+    connection = DataSourceExtension.get_mysql_connection(connection_info)
+    df = connection.sql(dialect_sql).limit(10).to_pandas()
+    print("### Result ###")
```

Ruff (0.8.2)

`print` found

(T201)